qemu/tests/usb_host.py

```diff
@@ -75,14 +75,18 @@ def _get_usb_mount_point():
         dev = re.findall(r'\[(sd\w+)\]', o)[0]
         mounts_cmd = "cat /proc/mounts | grep /dev/%s" % dev
         s, o = session.cmd_status_output(mounts_cmd)
-        if s:
-            s, o = session.cmd_status_output('mount /dev/%s /mnt' % dev)
+        if not s:
+            s, o = session.cmd_status_output('umount /dev/%s' % dev)
             if s:
-                test.error("Fail to mount /dev/%s, output: %s" % (s, o))
-            mp = "/mnt"
-        else:
-            mp = re.findall(r'/dev/%s\d*\s+(\S+)\s+' % dev, o)[0]
-        return mp
+                test.error("Fail to umount /dev/%s, output: %s" % (s, o))
+        mkfs_cmd = "mkfs.vfat /dev/%s" % dev
```

Comment:
No matter the status of usb stick, make this step as must, clean all the existing data if there are ?

Reply:
Updated, thanks!

Reply:
Yes! The purpose of this PR is to umount the passthrough USB stick if it's already mounted, re-make the filesystem on the USB stick to prepare a clean test environment, and then mount it (again) to guest for the following r/w testing.

Reply:
ok, thanks!